### DISCUSSION TITLE
Limit the bot from showing in the not allowed channel

### DISCUSSION THREAD

**Discussion by mkagit** [jakobdylanc/llmcord.py#78] [General]
`allowed_channel_ids` limits the bot from working in another channel but this one. But how to limit it from showing as a bot username there? Maybe confusing to call the bot and not getting answer.

**Comment by jakobdylanc** [comment]
You can accomplish this by editing the bot's permissions in your Discord server like you would any other user.

Specifically, make it so the bot doesn't have permission to access certain channels. Then the bot's @ won't appear in those channels.
